kubernetes/schema_pod_spec.go

```diff
@@ -292,6 +292,16 @@ func podSpecFields(isUpdatable, isComputed bool) map[string]*schema.Schema {
 							Schema: seLinuxOptionsField(isUpdatable),
 						},
 					},
+					"fs_group_change_policy": {
+						Type:        schema.TypeString,
+						Description: "fsGroupChangePolicy defines behavior of changing ownership and permission of the volume before being exposed inside Pod. This field will only apply to volume types which support fsGroup based ownership(and permissions). It will have no effect on ephemeral volume types such as: secret, configmaps and emptydir.",
```

Comment:
```suggestion
						Description: "fsGroupChangePolicy defines the behavior of changing ownership and permission of the volume before being exposed inside Pod. This field will only apply to volume types which support fsGroup-based ownership (and permissions). It will have no effect on ephemeral volume types such as: secret, configmaps and emptydir.",
```

Reply:
TBH, this is a copy/paste of the k8s documentation as all of the other doc string. As this seems to be the idiom for the documentation of the option, I'll let it like that